client-app/src/libs/shopping-cart/views/add-to-order-draft-button/add-to-order-draft-button.ts

```diff
@@ -31,15 +36,13 @@ export default class AddToDraftButton extends Vue {
   addItemToDraft!: (criteria: AddCartItem) => void;
 
 
-  async addToNewDraft() {
+  async addToNewDraft(newDraft: AddDraft) {
     (this.$refs.dropdown as any).hide();
-    const addNewDraft = new AddDraft();
-    addNewDraft.listName = this.newDraftName;
-    await this.addNewDraft(addNewDraft);
+    await this.addNewDraft(newDraft);
     const addItemToDraft = new AddCartItem();
```

Comment:
a little bit strange name for entity.. could we use the more clear name? Eg: "CartItem"